docs/js-core-1/week-1/mentors.md

```diff
@@ -16,6 +16,16 @@ You can find the preparation material [here](../preparation).
 
 Be sure to assign all of the tasks to your students several days before you aim to teach.
 
+## Delivery Notes
+
+### Giving Feedback to Each Other
+
+For this exercise, you should open the Pull Request and share you screen. Illicit answers from the students based on what they've just read.
```

Comment:
```suggestion
For this exercise, you should open the Pull Request and share you screen. Elicit answers from the students based on what they've just read.
```

Reply:
Ha! I blame your Github handle for this one!